spec/traits.dd

```diff
@@ -411,6 +413,40 @@ static assert(!__traits(isTemplate,foo!int()));
 static assert(!__traits(isTemplate,"string"));
 ---
 
+$(H2 $(GNAME isModule))
+
+        $(P Takes one argument. If that argument is a symbol that refers to a
+        $(DLINK spec/module, module) then $(D true) is returned, otherwise $(D false).
+        $(DDSUBLINK spec/module, package-module, Package modules) are considered to be
+        modules even if they have not been directly imported as modules.
+        )
```

Comment:
```
/dev/shm/dtest/work/repo/dlang.org/web/spec/traits.html:592: [UNDEFINED MACRO]         <!--UNDEFINED MACRO: "DLINK"--> then <span class="d_inlinecode donthyphenate notranslate">true</span> is returned, otherwise <span class="d_inlinecode donthyphenate notranslate">false</span>.
/dev/shm/dtest/work/repo/dlang.org/web/spec/traits.html:612: [UNDEFINED MACRO]         <!--UNDEFINED MACRO: "DLINK"--> then <span class="d_inlinecode donthyphenate notranslate">true</span> is returned, otherwise <span class="d_inlinecode donthyphenate notranslate">false</span>.
/dev/shm/dtest/work/repo/dlang.org/web/dlangspec.html:20849: [UNDEFINED MACRO]         <!--UNDEFINED MACRO: "DLINK"--> then <span class="d_inlinecode donthyphenate notranslate">true</span> is returned, otherwise <span class="d_inlinecode donthyphenate notranslate">false</span>.
/dev/shm/dtest/work/repo/dlang.org/web/dlangspec.html:20869: [UNDEFINED MACRO]         <!--UNDEFINED MACRO: "DLINK"--> then <span class="d_inlinecode donthyphenate notranslate">true</span> is returned, otherwise <span class="d_inlinecode donthyphenate notranslate">false</span>.
4 errors found. Exiting.
```

Things can use:

- `$(RELATIVE_LINK2 module, mytext)`
- `$(DDSUBLINK spec/module, module_declaration, mytext)`